### PR TITLE
DEEEPpppppLLLLLOOOOOYYYYEEEEddduuuhhh

### PR DESCRIPTION
**Pull request by hannahmanfredi**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
-FROM node:16.3.0
+FROM node:10.15.3
```
Comment (kimberlyjk) [inline Dockerfile:1]: lol giggling at the downgrade, but smart!

File: package.json [hunk 2]
```diff
@@ -18,11 +18,14 @@
...
   },
   "dependencies": {
+    "aws-sdk": "^2.922.0",
     "axios": "^0.21.1",
     "body-parser": "^1.19.0",
     "browser-sync": "^2.26.14",
     "cors": "^2.8.5",
     "express": "^4.17.1",
+    "grunt-aws-s3": "^2.0.2",
+    "grunt-s3": "^0.2.0-alpha.3",
```
Comment (kimberlyjk) [inline package.json:29]: interested to see how these work

File: server/server.js [hunk 3]
```diff
@@ -6,30 +6,32 @@ const cors = require('cors');
...
+  console.log('inside /booking')
   let campId = parseInt(req.query.campId);
   if (!campId) {
     campId = 0;
   }
   let init = {};
   const booked = await db_helper.findBookedArray({ campId: campId });
   init.booked = booked.booked;
-  await axios.get('http://localhost:3003/overview/pricing', { params: { campId: campId } })
+  await axios.get(`${routes.overview}/pricing`, { params: { campId: campId } })
```
Comment (kimberlyjk) [inline server/server.js]: where does` routes ` get defined?

**Review by kimberlyjk — APPROVED** [review]
Looks good! congrats on getting deployed. I know you were working hard on this. 

There is one area which I am concerned/confused. I commented on it but in your server.js file, you call a variable routes.overview and I don't see where in that file that variabel gets initialized/assigned info. you may want to look into that. 

overall, clean and concise!

File: package.json (hunk 2 above)
Comment (hannahmanfredi) [reply package.json:29]: lol so am I, haven't figured it out...yet

File: server/server.js (hunk 3 above)
Comment (hannahmanfredi) [reply server/server.js]: good catch..fixed now

**Pull request merged**